shell/assets/translations/en-us.yaml

```diff
@@ -1638,8 +1638,8 @@ cluster:
       title: Additional Kubelet Args
       machineSelector:
         label: Add Machine Selector
-        listLabel: Add Argument
-        bannerLabel: 'Note: The last selector that matches wins and only args from it will be used.  Args from other matches above will not combined together or merged.'
+        listLabel: Add Args
```

Comment:
I've changed this iteration of "Argument" to "Args" - this is the only one I reversed the decision on. I'd prefer to use "arguments" (see below) in this section. @jillian-maroket will happily take your advise on this one!

Reply:
@jillian-maroket the only thing I'll ask you to weigh in on for this PR is this comment in particular. There's only a handful of mentions. I think this should be changed to "arguments".

Reply:
@asettle Got it. We should use the full term "arguments". I don't think character length is an issue because there are longer strings on the UI.

Reply:
🫡 on it, boss